test/unit/api.spec.js

```diff
@@ -1,4 +1,5 @@
 import { standardizeTransactions } from '@/api'
+import { describe, expect, test } from 'vitest'
```

Comment:
if you don't want to write this line in every file we can set `globals` to `true`: https://vitest.dev/config/#globals